src/apps/bm_devkit/bmdk_common/app_main.cpp

```diff
@@ -396,7 +396,7 @@ static void defaultTask(void *parameters) {
                                  .sensorsPollIntervalMs = sys_cfg_sensorsPollIntervalMs};
   sensorSamplerInit(&sensorConfig);
   // must call sensorsInit after sensorSamplerInit
-  sensorsInit();
+//  sensorsInit();
```

Comment:
Should uncomment this. We don't want to turn off the sensors for all the other apps. Unless this is added somewhere else, which I am not currently finding.
```suggestion
sensorsInit();
```

Reply:
➕